org.opent2t.sample.binaryswitch.superpopular/org.opent2t.sample.binaryswitch.superpopular.json

```diff
@@ -0,0 +1,22 @@
+{
+  "id": "http://schemas.microsoft.com/org.opent2t.sample.binaryswitch.superpopular#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "",
+  "title": "OpenT2T Switch",
+  "definitions": {
+    "org.opent2t.sample.binaryswitch.superpopular": {
+      "type": "object",
+      "properties": {
+        "power": {
+          "$ref": "oic.r.switch.binary.json#/definitions/oic.r.switch.binary"
```

Comment:
Do we want to do the same thing with oic.r.switch.binary that we have with the other schemas from oneiota (e.g. oic.r.temperature) and include it in the repo for reference?